packages/stargate/src/signingstargateclient.ts

```diff
@@ -209,6 +214,26 @@ export class SigningStargateClient extends StargateClient {
     return this.signAndBroadcast(senderAddress, [sendMsg], fee, memo);
   }
 
+  public async cancelUnbondingDelegation(
```

Comment:
Let's not add more of those helper functions. This is a pretty unsustainable design from long time ago. We have way too many message types these days. A user can just use `signAndBroadcast`.